EndlessSky.xcodeproj/project.pbxproj

```diff
@@ -978,7 +978,7 @@
 			);
 			runOnlyForDeploymentPostprocessing = 0;
 			shellPath = /bin/sh;
-			shellScript = "${SRCROOT}/utils/set_dylibs_rpath.sh";
+			shellScript = "\"${SRCROOT}/utils/fetch_sdl2_framework.sh\"";
```

Comment:
This doesn't just add quotes, but also changes the script that gets called. Was that intentional?

Reply:
!!! no, thank you for spotting! That might fix this too!